### PR TITLE
Various fixes before release 1.1.4

### PR DESCRIPTION
**Pull request by bmarty**
Please review commit by commit

### CODE REVIEW

File: vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt [hunk 1]
```diff
@@ -1675,10 +1675,12 @@ class RoomDetailFragment @Inject constructor(
             shareText(requireContext(), action.messageContent.body)
         } else if (action.messageContent is MessageWithAttachmentContent) {
             lifecycleScope.launch {
```
Comment (ganfra) [inline vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt:1677]: why not using viewLifecycleScope? isAdded check would not be needed then

File: vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt [hunk 2]
```diff
@@ -1701,16 +1703,22 @@ class RoomDetailFragment @Inject constructor(
...
-                saveMedia(
-                        context = requireContext(),
-                        file = data,
-                        title = action.messageContent.body,
-                        mediaMimeType = action.messageContent.mimeType ?: getMimeTypeFromUri(requireContext(), data.toUri()),
-                        notificationUtils = notificationUtils
-                )
-            }
+            val result = runCatching { session.fileService().downloadFile(messageContent = action.messageContent) }
+            if (!isAdded) return@launch
```
Comment (ganfra) [inline vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt:1707]: Same remark ^

**Review by ganfra — APPROVED** [review]
2 small remarks otherwise LGTM

File: vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt (hunk 1 above)
Comment (bmarty) [reply vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt:1677]: yeah, we will change that later, I'm not even sure if this isAdded statement is still necessary with `lifecycleScope`

**Pull request merged**